### DISCUSSION TITLE
Header of the file

### DISCUSSION THREAD

**Discussion by alexisInSAR** [insarlab/MintPy#992] [General]
Hi all, 

Is it normal to have a deletion in the header of MintPy scripts: i.e., "#/usr/bin/env python3" does not appear on the first line of the scripts? 

Thank in advance, 

Kind regards, 

Alexis

**Comment by yunjunz** [comment]
Hi Alexis, this change is introduced in #828, and yes, it's intentional: the scripts within `mintpy.cli` sub-module are now the executables with the `#!/usr/bin/env python3` header, whereas the original script, e.g. `mintpy.view`, is served only as a Python sub-module. You may see the file permission is adjusted according to this as well. 

We are aware that this is a backward-incompatible change. I put a migration note here (https://github.com/insarlab/MintPy/discussions/913). If you have more specific questions or issues with this change, please do not hesitate to let me know.

**Reply by alexisInSAR** [reply]
Hi, 

Thank you for your reply. I will make the necessary changes to make my scripts compatible with MintPy. 

Kind regards, 

Alexis
